src/index.ts

```diff
@@ -18,12 +19,17 @@ export const run = () => {
       }
     });
 
-    if (!args[0] || !AVAILABLE_COMMANDS.includes(args[0])) {
-      showHelp(args[0], options.help);
+    if (!args[0] || !AVAILABLE_COMMANDS.includes(args[0]) || args.length > 2) {
+      showHelp(args, options.help);
     } else if (args[0] === 'android') {
-      const androidSetup = new AndroidSetup(options);
-      androidSetup.run();
-    } else {
+      if (args[1]) {
+        const sdkCommandExecute = new SdkCommandExecute(args[1], options);
```

Comment:
Maybe rename the class to `AndroidSubcommand`? And so the next line would be `androidSubcommand.run()`.